front/components/wysiwyg/editor.ts

```diff
@@ -0,0 +1,19 @@
+import Document from "@tiptap/extension-document";
+import { HardBreak } from "@tiptap/extension-hard-break";
+import { History } from "@tiptap/extension-history";
+import Paragraph from "@tiptap/extension-paragraph";
+import Text from "@tiptap/extension-text";
+
+export const CONTENT_EDITOR_EXTENSIONS = [
+  Document,
+  Text,
+  Paragraph,
+  History,
+  HardBreak.extend({
```

Comment:
No need for hardbreak, you can override shortcuts for `Paragraph`. Also adding `Hardbreak` would have required to update how we convert Tip Tap nodes to raw text.